docker/sample-damap-realm-export.json

```diff
@@ -701,8 +701,12 @@
       "enabled": true,
       "alwaysDisplayInConsole": false,
       "clientAuthenticatorType": "client-secret",
-      "redirectUris": [],
-      "webOrigins": [],
+      "redirectUris": [
+        "http://localhost:8085/*"
+      ],
+      "webOrigins": [
+        "*"
```

Comment:
I think webOrigins could be "+" here, but since its a example it doesn't count that much, but others could take this example realm to bootstrap other environments.